server/server/src/discovery.rs

```diff
@@ -11,7 +11,7 @@ const HARDWARE_ID_KEY: &str = "hardware_id";
 const CLIENT_VERSION: &str = "unspecified";
 
 pub(crate) fn start_discovery(protocol: Protocol, port: u16, hardware_id: String) {
-    tokio::task::spawn(async move {
+    tokio::spawn(async move {
```

Comment:
This one was not required change, i just wanted to align with other (did a global search for spawn)